buttons/buttons.css

```diff
@@ -1310,3 +1310,46 @@ border-radius: 25px;
   right: 0;
 }
 
+
+.ankush-btn-1 {
+	width: 170px;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	text-align: center;
+	color: #000;
+	text-transform: uppercase;
+	font-weight: 600;
+	margin-left: 30px;
+	margin-bottom: 30px;
+	cursor: pointer;
+	display: inline-block;
+	background-color: transparent;
+	border: 3px solid #00d7c3;
+	border-radius: 50px;
+	transition: all 0.15s ease-in-out;  
+	color: black;
+	background-image: linear-gradient(30deg, #00d7c3 50%, transparent 50%);
+	background-size: 500px;
+	background-repeat: no-repeat;
+	background-position: 0%;
+  }
+  .ankush-btn-1:hover {
+	color: white;
+	background-color: black;
+	box-shadow: 0 0 10px 0 #00d7c3 inset, 0 0 20px 2px #00d7c3;
+	border: 3px solid #00d7c3;
+	animation: pulse 1s infinite;
+	transition: 0.3s;
+	background-position: 100%;
+  }
+  @keyframes pulse {
```

Comment:
can u pls rename thins to some thing like yourname-index-pulse or yourname-surname-pulse or yourname-btn-{your button index}-pulse
cause it is already present in file and it will overwrite existing animation